include/global.h

```diff
@@ -169,7 +169,11 @@ struct UCoords32
 
 struct SaveBlock3
 {
-};
+#if USE_DEXNAV_SEARCH_LEVELS == TRUE
+    u8 dexNavSearchLevels[ROUND_BITS_TO_BYTES(NUM_SPECIES)];
+#endif
+    u8 dexNavChain;
```

Comment:
Oh, I don't think I realize that this was going in the saveblock. I added `gChainFishingDexNavStreak` in `src/wild_encounters.c` assuming we could share them later. Should we merge them?